package.json

```
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "prebuild": "shx rm -rf dist && yarn generate",
+    "postbuild": "npx dependency-check package.json --missing ./dist/**/*.js",
```

Avoid the use of dev dependencies

```suggestion
    "postbuild": "npx dependency-check package.json --no-dev --missing ./dist/**/*.js",
```

---

using _--no-dev_ we see another dependency: _danger-plugin-digitalcitizenship_ 
This is used within Dangerfile.js
Is it correct to have this file in _dist/_ fodler?